packages/keys/src/models/index.ts

```diff
@@ -1,12 +1,14 @@
-export interface KeyPair {
-    privateKey?: string;
-    publicKey?: string;
-}
-
 export const ECDSA_PATTERNS = {
   secp256k1: {
     PRIVATE_KEY: /^[a-f0-9]{64}$/,
     PUBLIC_KEY: /^[a-f0-9]{66}$/,
     SIGNATURE: /^[a-f0-9]{128}$/,
   },
 };
+
+/**
+ * Algorithms specifies, which algorithm has to be used with a particular public key
+ */
+export enum Algorithms {
```

Comment:
Can we change this to `KeyTypes`?
```suggestion
/**
 * KeyType specifies which curve (for elliptic curve keys) or algorithm (e.g. RSA) a key is associated with 
 */
export enum KeyType {
```